Bio/Seq.py

```diff
@@ -1139,7 +1140,7 @@ def ungap(self, gap=None):
 
 
 class UnknownSeq(Seq):
-    """A read-only sequence object of known length but unknown contents.
+    """Create a read-only sequence object of known length but unknown contents.
```

Comment:
See comment for the ``Seq`` class docstring, ``Create ...`` is not quite right here.